src/client.ts

```diff
@@ -316,20 +316,26 @@ export class PluggyClient extends BaseApi {
    * - headers (optional): The headers to send with the webhook
    * @returns {Webhook} the created webhook object
    */
-  async createWebhook(webhookParams: CreateWebhook): Promise<Webhook> {
```

Comment:
since CreateWebhook is no longer used we can remove it?

Reply:
Removed, thanks!